Cargo.toml

```diff
@@ -1,3 +1,25 @@
+[workspace]
+resolver = "2"
+members = [
+  "benches/benchsuite",
+  "benches/capture",
+  "crates/cargo-platform",
+  "crates/cargo-test-macro",
+  "crates/cargo-test-support",
+  "crates/cargo-util",
+  "crates/crates-io",
+  "crates/credential/*",
+  "crates/home",
+  "crates/mdman",
+]
+exclude = [
+  "crates/resolver-tests",
+  # For linkchecker (downloaded during CI) and semver-check
+  # TODO: consider move semver-check to crates/ folder
```

Comment:
Does the `linkcheck.sh` script need to be updated to support running from a directory separate from the book?